### ISSUE TITLE
Update pre-check-in text message

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#56176]
## Description 

How might we update the pre-check-in text message to, 1) remove reference of "pre-check-in,", 2) align with the unified check-in experience wireframes and content, and 3) avoid telling the Veterans to "confirm their appointment" more than once. ([Stephen made reference to this here](https://dsva.slack.com/archives/C022AC2STBM/p1680529940019739?thread_ts=1678390007.447659&cid=C022AC2STBM).) 

Note: A [separate ticket](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/53569) is being worked on for text message content for check in. 

## Tasks

- [x] Update content for pre-check-in text message. [Existing message is found here](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/DEP0lnk#Comment). 
- [ ] Review with sitewide content at office hours. 
- [ ] Update the text message wireframe in sketch. 

## AC

- [ ] Reviewed with sitewide content at office hours. 
- [ ] Added into sketch unified experience file. 
- [ ] Reviewed with UX and product. 
- [ ] Reviewed by Stephen.

**Comment by benbrasso-agile6** [comment]
@cgednalske @kellysmith1008 want to do some async on this one? 

I created a couple variations yesterday for the text message to get some ideas going: 

https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/DEPQpLj
https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/j4gLwbm

What ya'll think? What else should we try? 

I think it would be a good idea to stop into content office hours tomorrow to get a quick reaction.

**Comment by benbrasso-agile6** [comment]
I'm going to close this. Waiting on feedback from sitewide content.
